src/repl-jline/scala/tools/nsc/interpreter/jline/JLineHistory.scala

```diff
@@ -15,7 +15,7 @@ import scala.tools.nsc.interpreter
 import scala.tools.nsc.interpreter.session.{History, SimpleHistory}
 
 
-/** A straight scalification of the jline interface which mixes
+/** A straight scalafication of the jline interface which mixes
```

Comment:
With the *a* instead of *i* does seem to be the established choice based on a couple of Google searches.